wikipedia-sentences/README.md

```diff
@@ -62,12 +62,47 @@ To quickly get started, you can index a [small dataset of 50 sentences](data/toy
 ```sh
 python app.py -t index
 ```
-If you see the following output, it means your data has been correctly indexed. 
+The relevant Jina code to index data given your flow's YAML definition breaks down to
+```python
+with Flow().load_config('flows/index.yml'):
+    f.index_lines(filepath='data/toy-input.txt', read_mode='r', batch_size=16, num_docs=10)
+```
+The flow will interpret each line in the txt file as one Document.
```

Comment:
Flow

Reply:
Fixed in https://github.com/jina-ai/examples/pull/577/commits/fd98b7492d5df999878f1f3526b9fce2ba3f8943